### DISCUSSION TITLE
How to make image work from PC

### DISCUSSION THREAD

**Discussion by xtheeq** [moudey/Shell#493] [Q&A] [answered]
`item(title="wezterm" tip=tip_run_admin admin=has_admin image="C:\Users\{username}\Documents\logo\wezterm.png" cmd='wezterm.exe' args='start --cwd "@sel.dir"')`

**Comment by xtheeq** [comment, marked as answer]
shall use double slash windows thing

**Comment by moudey** [comment]
```js
image='C:\Users\@user.name\Documents\logo\wezterm.png'
```
